lesson2.html

```diff
@@ -178,7 +178,6 @@ <h3>String</h3>
 					}
 					</code></pre>
 					<p>Notice that we can use + to append/glue Strings to each other.</p>
-					<p>This works for other types too, Java does the conversion to String for us.</p>
```

Comment:
Just a note about this - I think that since we are trying to stress that types are individual and you have to stick with them, perhaps introducing the idea of conversion here is not wise. What do you think?

Reply:
coolio, yeah its something they dont need to necessarily think about